diff --git a/node_modules/ipfs-unixfs-importer/dist/src/utils/persist.js b/node_modules/ipfs-unixfs-importer/dist/src/utils/persist.js
index 50202bf..675533f 100644
--- a/node_modules/ipfs-unixfs-importer/dist/src/utils/persist.js
+++ b/node_modules/ipfs-unixfs-importer/dist/src/utils/persist.js
@@ -1,11 +1,14 @@
 import * as dagPb from '@ipld/dag-pb';
 import { CID } from 'multiformats/cid';
 import { sha256 } from 'multiformats/hashes/sha2';
+import { blake2b256 } from "@multiformats/blake2/blake2b";
 export const persist = async (buffer, blockstore, options) => {
     if (options.codec == null) {
         options.codec = dagPb;
     }
-    const multihash = await sha256.digest(buffer);
+    console.log("POLO")
+
+    const multihash = await blake2b256.digest(buffer);
     const cid = CID.create(options.cidVersion, options.codec.code, multihash);
     await blockstore.put(cid, buffer, options);
     return cid;
diff --git a/node_modules/ipfs-unixfs-importer/src/utils/persist.ts b/node_modules/ipfs-unixfs-importer/src/utils/persist.ts
index 2911b04..ddbf5c6 100644
--- a/node_modules/ipfs-unixfs-importer/src/utils/persist.ts
+++ b/node_modules/ipfs-unixfs-importer/src/utils/persist.ts
@@ -1,6 +1,7 @@
 import * as dagPb from '@ipld/dag-pb'
 import { CID } from 'multiformats/cid'
 import { sha256 } from 'multiformats/hashes/sha2'
+import { blake2b256 } from "@multiformats/blake2/blake2b";
 import type { WritableStorage } from '../index.js'
 import type { Version as CIDVersion } from 'multiformats/cid'
 import type { BlockCodec } from 'multiformats/codecs/interface'
@@ -17,7 +18,8 @@ export const persist = async (buffer: Uint8Array, blockstore: WritableStorage, o
     options.codec = dagPb
   }
 
-  const multihash = await sha256.digest(buffer)
+  console.log("Using blake2b")
+  const multihash = await blake2b256.digest(buffer)
   const cid = CID.create(options.cidVersion, options.codec.code, multihash)
 
   await blockstore.put(cid, buffer, options)
